app/src/common/services/notification/notificationService.ts

```diff
@@ -6,7 +6,14 @@ namespace common.services.notification {
 
         private toastOptions:any = {};
 
-        constructor(private message:string, private $mdToast:ng.material.IToastService, private $rootScope:global.IRootScope) {
+        private timeOut:number = 0;
```

Comment:
This should actually be initialised as undefined (don't provide an initialiser), as it will be valid to pass 0 (In angular `$timout(fn, 0)` is a common trick to delegate the function being run until a full digest has been run)